posts/seurat-cosmx-basics/index.qmd

```diff
@@ -0,0 +1,244 @@
+author:
+  - name: Claire Williams
+    affiliations:
+      - ref: nstg
+      - ref: crwilliams11
+affiliations:
+  - id: nstg
+    name: NanoString Technologies, Inc.
+    city: Seattle
+    state: WA
+  - id: crwilliams11
+    name: "Github: [@crwilliams11](https://github.com/crwilliams11)"
+toc: true
+toc-title: Contents
+toc-depth: 3
+toc-expand: 2
+toc-location: left
+number-sections: true
+number-depth: 4
+date: "2024-05-10"
+date-modified: last-modified
+categories: [visualization]
+draft: false
+---
+
+# Introduction 
+
+This is a non-comprehensive vignette to introduce the use of the Seurat package to visualize and work with data generated from the CosMx&#8482; Spatial Molecular Imager (SMI). This vignette does not cover analysis of data in Seurat but rather tries to address frequently asked questions we've received from customers on getting started with their data in Seurat. As additional resources, please see vignettes on the Seurat website: [Spatial Vignette](https://satijalab.org/seurat/articles/seurat5_spatial_vignette_2.html#human-lung-nanostring-cosmx-spatial-molecular-imager) and
+[Clustering Tutorial](https://satijalab.org/seurat/articles/pbmc3k_tutorial). Also see additional blog posts on our [scratch space](https://nanostring-biostats.github.io/CosMx-Analysis-Scratch-Space/about.html).
+
+For this vignette, we use a Seurat object made from the mouse brain public data set and assembled in the structure used by the Technology Access Program [TAP](https://nanostring.com/products/cosmx-spatial-molecular-imager/technology-access-program/); similar outputs are available from the AtoMx&#8482; Spatial Informatics Portal (SIP). To download raw data for this dataset, go [here](https://nanostring.com/products/cosmx-spatial-molecular-imager/ffpe-dataset/cosmx-smi-mouse-brain-ffpe-dataset/). 
+
+Like other items in our [CosMx Analysis Scratch Space](https://nanostring-biostats.github.io/CosMx-Analysis-Scratch-Space/about.html){target="_blank"},
+the usual [caveats](https://nanostring-biostats.github.io/CosMx-Analysis-Scratch-Space/about.html){target="_blank"} and [license](https://nanostring-biostats.github.io/CosMx-Analysis-Scratch-Space/license.html){target="_blank"} applies.
+This post will show you how to:
+
+- @sec-loading Load needed libraries and example dataset
+- @sec-data-structure Find important data in Seurat object
+- @sec-spatial-plotting Plot CosMx data in space
+- @sec-dim-reduction-plots Visualize dimension reduction data 
+
+
+
+
+# Data Loading {#sec-loading}
+Here we start from a Seurat object containing CosMx data and analysis stored in an rds file. To start from raw data instead, see the Seurat Spatial Vignette above.
+
+First, load needed libraries:
+```{r load_libraries, message=FALSE, eval = FALSE}
+library(Seurat)
+library(ggplot2)
+```
+
+Load in the Seurat object, available on Box.com [here](https://nanostring.box.com/s/y7ielc54qw1g1f5x98hxczdo8tezobbg).
```

Comment:
When possible, we should configure the hyperlinks to open an new tab so that we can keep the blog tab open. To do this, add {target="_blank"} after the () in the link like this: 
> ```Load in the Seurat object, available on Box.com [here](https://nanostring.box.com/s/y7ielc54qw1g1f5x98hxczdo8tezobbg){target="_blank"}```

Reply:
I didn't know about this formatting, thanks! All links now use this configuration as of commit [ef85653](https://github.com/Nanostring-Biostats/CosMx-Analysis-Scratch-Space/pull/29/commits/ef85653ce89530455af85b971532a3b02b02e08e)